runtime/IO/File.h

```diff
@@ -10,7 +10,9 @@
 #include <Jakt/Error.h>
 #include <Jakt/RefCounted.h>
 #include <Jakt/String.h>
+#include <dirent.h>
 #include <stdio.h>
+#include <sys/stat.h>
```

Comment:
you need to guard these additions with `#ifndef _WIN32`, because on Windows, mkdir comes from `<direct.h>`

But more than that, the includes should go into the .cpp file because they are only used  in the implementation. Nothing in the public interface needs either of these. 

https://docs.microsoft.com/en-us/cpp/c-runtime-library/reference/mkdir?view=msvc-170>
https://docs.microsoft.com/en-us/cpp/c-runtime-library/reference/mkdir-wmkdir?view=msvc-170

```c++
#ifdef _WIN32
#    include <direct.h>
#else
#    include <dirent.h>
#    include <sys/stat.h>
#endif
```